### DISCUSSION TITLE
Serializers

### DISCUSSION THREAD

**Discussion by AbhijithGanesh** [CiviWiki/OpenCiviWiki#1166] [Q&A]
@brylie I thought we used DRF for serialization and it had been in use for all apps, wasn't that the case?

**Comment by brylie** [comment]
There may be inconsistent serialization throughout the API code. 

Since we are transitioning to the conventional Django Model-Template-View paradigm, we should de-emphasize API development for the time being. That way, we can focus our limited resources on the Backbone to Django template/view migration.

**Reply by AbhijithGanesh** [reply]
I agree that we should prioritize but do you believe that current standards for serialization isn't the best and we may have to improve it in the near future?

**Reply by brylie** [reply]
We should follow Django REST Framework serialization conventions in the future.
